stubs/psycopg2/psycopg2/pool.pyi

```diff
@@ -9,14 +9,11 @@ class AbstractConnectionPool:
     maxconn: Any
     closed: bool
     def __init__(self, minconn, maxconn, *args, **kwargs) -> None: ...
-
-class SimpleConnectionPool(AbstractConnectionPool):
-    getconn: Any
-    putconn: Any
-    closeall: Any
-
-class ThreadedConnectionPool(AbstractConnectionPool):
-    def __init__(self, minconn, maxconn, *args, **kwargs) -> None: ...
+    # getconn, putconn and closeall are officially documented as methods of the
+    # abstract base class, but in reality, they only exist on the children classes
     def getconn(self, key: Any | None = ...): ...
-    def putconn(self, conn: Any | None = ..., key: Any | None = ..., close: bool = ...) -> None: ...
+    def putconn(self, conn, key: Any | None = ..., close: bool = ...) -> None: ...
```

Comment:
CI requested that I removed the default value for the `conn` argument. This is in line with the documentation so I applied it.